docs/roadmap.md

```diff
@@ -6,51 +6,41 @@ The roadmap for Project Reunion supports the primary goals of breaking down the
 
 For a general overview, see the [readme](https://github.com/microsoft/ProjectReunion/tree/master/docs) and [FAQ](https://github.com/microsoft/ProjectReunion/blob/master/docs/faq.md).
 
-Here's how we intend to roll out Project Reunion *(all times/features/releases are tentative and subject to change)*:
-
-<table>
-  <tbody>
-    <tr>
-      <th>Reunion 0.5 (Q1 2021)</th>
-      <th>Reunion 0.8 (Q2 2021)</th>
-      <th>Reunion 1.0 (Q3 2021)</th>
-      <th>2022 and Beyond</th>
-    </tr>
-    <tr>
-    <td>
-      <ul>
-            <li>1st supported public release</li>
-            <li>WinUI 3 works in <a href="https://docs.microsoft.com/en-us/windows/msix/desktop/desktop-to-uwp-packaging-dot-net">Desktop Packaged (Win32)</a> Apps</li>
-            <li>Win2D</li>
-            <li>DwriteCore</li>
-            <li>MRTCore</li>
-       </ul>
-     </td>
-     <td>
-        <ul>
-            <li>WinUI improvements</li>
-            <li>Framework Package support for Unpackaged apps</li>
-            <li>App Lifecycle API</li>
-            <li>State Notifications</li>
-         </ul>
-     </td>
-     <td>
-        <ul>
-            <li>Reunion Windowing support</li>
-            <li>New Rendering and Input features</li>
-            <li>Smooth WinUI input & composition system experience</li>  
-         </ul>
-     </td>
-    <td>
-        <ul>
-            <li>Multiple releases per year</li>
-            <li>Continue expanding existing app compatibility</li>
-            <li>Continuous delivery of new easier to use capabilities</li>
-         </ul>
-     </td>
-    </tr>
-  </tbody>
-</table>
+
+Here's how we intend to roll out Project Reunion *(all times/features/releases are tentative and subject to change)*: 
+
+✅ = Supported
+
+❌ = Not Supported 
+
+🔄 = Preview 
+
+#### Packaged Desktop Apps
+
+ 🌟 | Reunion 0.5 (Q1 2021) | Reunion 0.8 (Q2 2021) | Reunion 1.0 (Q3 2021) | 
+-- | --------------------- | --------------------- | --------------------- |
+WinUI |                ✅ |                    ✅ |                   ✅ | 
+DWriteCore |           ✅ |                    ✅ |                   ✅ | 
+MRT Core |             ✅ |                    ✅ |                   ✅ | 
+App Lifecycle APIs |   🔄 |                    🔄 |                   ✅ | 
+Power State Notifications |  ❌ |              🔄 |                   ✅ | 
+Windowing |            ❌ |                    ❌ |                   ✅ | 
+Deploy Options: |  
+• In-app |             ✅ |                    ✅ |                   ✅ | 
+• Fx Package |         ✅ |                    ✅ |                   ✅ | 
+
```

Comment:
I think we lost some information here.

1. WinUI in Reunion 0.5 was not supporting unpackaged scenarios, with that coming later in Reunion 0.8. Is that still the case? Perhaps a 🟨 is warranted here for WinUI in Reunion 0.5? Or 🔄?

2. WinUI in Reunion 0.8 was slated to support App Lifecycle APIs and Power State Notifications. Now both show as 🔄 Preview only. Was this pushed back? Or just an old roadmap bug?

Reply:
Thanks Rafael! Let me know if these answer your questions

1. WinUI's unpackaged support is slated to come in 1.0 as the table shows below (everything here is still tentative though, no guarantees). As far as I can see, our previous roadmap never stated when WinUI support for unpackaged apps would be coming (unless there's another roadmap I'm not looking at).

![image](https://user-images.githubusercontent.com/13246069/108137838-8b8fbb00-7071-11eb-8353-6b94ac8ba756.png)

2. Our previous roadmap did show App Lifecycle and Power State for 0.8, but the previous roadmap did not distinguish between fully supported features and preview features. This updated roadmap is much clearer both in what to expect depending on your app type, and what to expect in terms of a preview vs supported feature.

Reply:
Sounds good, thanks!